analyzer/main.zeek

```diff
@@ -284,6 +284,16 @@ function set_session(c: connection, message_id: int, opcode: LDAP::ProtocolOpcod
 }
 
 #############################################################################
+@if (version::at_least("6.0.0"))
```

Comment:
This has landed in zeek-5.2.0, let's just make that switch with that version.

Also, small spelling correction.

```suggestion
@if (Version::at_least("5.2.0"))
```